### PR TITLE
[TR] PIM-4093: rename the base catalog storage filesystem

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                  | A
| ------------------ | ---
| Migration script?  | -
| Behats             | Y
| Specs & Static     | Y
| Changelog updated  | -

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@ app/export/*
 app/emails/*
 app/uploads/*
 app/uploads/product/*
+app/file_storage/*
```
Comment (sumbobyboys) [inline .gitignore]: :+1:

Comment (sumbobyboys) [inline .gitignore]: don't forget to add it in standard and ee too :)

**Comment by jmleroux** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**